layers/+lang/bibtex/packages.el

```diff
@@ -28,7 +28,9 @@
     (ivy-bibtex :requires ivy)
     markdown-mode
     org
-    org-ref))
+    org-ref
+    biblio
```

Comment:
@arifer612 Thanks for the review. I deliberately chose to install `biblio` by default. But just as for all other packages, I refer to its docs for further instructions. I did not really emphasize that is gets installed (I just added it to the list of 'package references') because I think much of its functionality overlaps with the 'org-ref' package (I did not study in detail the features of `org-ref` and `biblio`). But for example, `biblio` provides a function to retrieve bibtex info by doi, which `org-ref` lacks, and as it is only a small package, I figured it does not hurt to install it by default and provide some extra useful features to the bibtex layer.

Also I assumed that in practice, most users like to import articles with a browser extension via JabRef, so I figured that users who really prefer to use biblio will find their way to its documentation (I already provided them with a link, which lacks for the other packages).

Reply:
`org-ref` also provides a means to retrieve bibtex info by doi through `doi-utils-add-bibtex-entry-from-doi` but from what I see, `biblio` does indeed provide more backends to search from like DBLP, HAL, and IEEE. The compatibility with browser extensions is a good read to include `biblio` as you mentioned. I think I'll look more into it then!

Reply:
Interesting, somehow I could not find that doi function (probably I searched in the org-ref namespace), but I had noticed also the extra backends.

Reply:
After comparing the bibliography organisation that `org-ref` and `ebib` provide, I've come to learn that whilst `ebib` does everything that `org-ref` provides with a much prettier interface except for cross referencing. It would be good if we could 'turn off' one while the other is selected.

Reply:
Although I understand your point about providing either the one or the other, I don't understand what you mean with mixing components. But despite your idea sounds sensible (if the packages really are mostly two similar alternatives), it still does not seem a big issue to me if org-ref always gets installed by default. If users really don't want it, then they could add it to `dotspacemacs-excluded-packages`.

It is a little too much for me to read the full docs of `org-ref` and `ebib`, but on first sight it seems that org-ref is focused on `org-mode` while ebib is more general.

My initial intention for this PR is just to offer an easy Spacemacsey configuration for `ebib` to Spacemacs users. Personally, I had a separate private `ebib` layer. But for Spacemacs I figured it would be 'cleaner' to offer it as an option in the bibtex layer.